### ISSUE TITLE
Need resource constrains for must-gather-api service

### ISSUE THREAD

**Issue by fbladilo** [konveyor/forklift-operator#169]
We need to supply resource contrains for the must-gather-api pod , essentially a set of default memory and cpu requests/limits for the service. @aufi @fabiendupont any hints/ideas on these?

**Comment by aufi** [comment]
Good point, the forklift-must-gather-api runs a tiny webserver and executes oc adm must-gather, both is not very demanding on memory and CPU. I'd consider 1GiB memory limit that should be more than okay (maybe 512MB could work too). The CPU single core limit and I'm not sure how it can be specified, but it is not expected to consume much CPU, so some low limit should be OK.

What could go higher is disk usage since all must-gather results are temporary stored on the pod, I'd suggest disk size limit to 10-20GB.

**Comment by fbladilo** [comment]
@aufi I think we can have a good start with just cpu/memory, I would try to avoid being too aggressive on these settings and cause unnecessary k8s scheduling issues due to constrains. The requests is the minimum we need in order to operate so we can go low there , the limits a bit trickier for cpu it will throttle usage at the limit but for memory it will cause restarts due to OOM. I'll try to see in a used cluster that runs must-gather-api requests and watch usage for an initial pass at these settings.
